### ISSUE TITLE
Feature Request: Automatic role handling in discord

### ISSUE THREAD

**Issue by itsthejoker** [GrafeasGroup/tor#165]
/u/Aaeriele: I haven't had reason to care about it yet, but perhaps the bot which handles claiming etc could also automatically announce when someone hits a transcription role threshold into #transcription_roles instead of people having to post screenshots

**Comment by DavidArchibald** [comment]
I'd be happy to have ToR-Stats do this so that another discord bot doesn't have to be made, all that would be required would be a go ahead signal and for ToR-Stats to be given access to #transcription_roles because he can't access it, and it really would be simple because he already announces gamma changes and roles, just in a different server.

**Comment by itsthejoker** [comment]
Yes, that would work. Let's work on getting ToR-Stats access to the API once it's done migrating, then we can have it post to Discord once we're using a single source of truth for the data.
